docs/suppressing-instrumentation.md

```diff
@@ -6,11 +6,100 @@ You can disable the agent using `-Dotel.javaagent.enabled=false`
 ## Suppressing specific agent instrumentation
 
 You can suppress agent instrumentation of specific libraries by using
-`-Dotel.instrumentation.[id].enabled=false`.
+`-Dotel.instrumentation.[name].enabled=false` where `name` is the instrumentation `name`
+(main or additional):
 
-where `id` is the instrumentation `id`:
-
-[TODO add table here with all instrumentation ids]
+| Instrumentation name |
+|---------------------------|
+| akka-actor|
+| akka-http|
+| apache-camel|
+| apache-dubbo|
+| apache-httpasyncclient|
+| apache-httpclient|
+| armeria|
+| async-http-client|
+| aws-lambda|
+| aws-sdk|
+| axis2|
+| cassandra|
+| classloader |
```

Comment:
This module was recently renamed to `internal` (and it probably shouldn't be on this list)
```suggestion
```

Reply:
I see three names: `internal-class-loader`, `internal-proxy` and `internal-url-class-loader`. Why do you think these should not be on the list (ie user should not be able to disable them)?

Reply:
Yes, they are required for agent, and should not be disabled.

Reply:
Because without them the javaagent won't function properly (e.g. `helperResourceNames()` won't get injected).

Reply:
Sounds right :)